### DISCUSSION TITLE
求激活码

### DISCUSSION THREAD

**Discussion by user2027** [LFTEC/LearningFucker#35] [General]
------------------------------License------------------------------
NzFFxJsBv12VbQ5bgpTTfE2UTeE4yOk8VuMdc6PkDp34q0qNivpEsSIZYTdT1dGyKnM
/B4MsDidYJKDE4gmSrLz6yA/3lzyPSoj8782pzrfDGWhJIpFuCfvTzDrDyJJk9K34n+
4M9E45P8PGCxtskkR9rAF8WFy9+ZUgT95KUYqoorL76avSAvmyYWZ9PZDSUSo6+uump
+FNd9Vyp8AujrfAVYaBf6ZeTyrnr1mgGjEJg9Rycxy2uxCkaQYOVFPy+lTzpNs8Zyaa
riVSXCN+p1xdED7K18Wz6nfK+XBI2Q7MHGaqZKrp0oEilRdDelUdMxx5bq/R+f17b5t
Gij8YX/E662kIPN1aUyGyzs1Nhorit57GKfzhj83VwN2/wXmvYe3Wook6+1Zg+l7Maw
2llq2oqlhN5ubNKYBDZW2lWHj9G82YqSU/NiHDxn4+ywd99fQhIv0SS+fPM/ayTS18d
09JXcPlamUubDierCk9xvOpyx0Sb1H8WPUCx+XhzBnKL+dKa8ydKk9XNtWNxZOjLsfj
BIvGNPe2esXylISZ+dbn5GPwv5Qm0IV4/kcbfm91YmavDMVcP2kfWdVNFczJFwqTX7W
T8DmeNQqNcneEvCmk4Ly3JoSHXyYiCnTRAjj9jReSba1zqHaNGBfjLd8dMEFABFQQW8
VxhBUdRO+d/xlLeOpjqtI+7J0KtKGSLAK0lsZpPgUdh3+sBkJIWDQilvJmiDk+2FoA5
DYO0lDwmK74PCKNOkDzI5jX6kOTLtoRSSdWX/AUAhC7zLE/tHF/TVz2is35C76YKBmv
D66wEoN898hD+QZwOqdDHZcAJR3qg2/SN0V4bX0RwYLwmb4EKzvpEH/3IETkT1UeSDl
+WagXOQ5MMLHOhmWO9ivbzukedAMR6DTZT1Ga2trrtxoHcESkTn9FWHHlhth3+pVI9O
7bgYFBSkOkBvqEprg9OuTC+49kBhdZ8p+uL2cBYqkXZDzFNDy+uFEwWOjdNdC9ELAZh
Hxz4vGGUaPNHAp3glV+PrtfOMIDFxLedJa4ZGGnFdtKh2cuWMmKXVSn6+qgqMCmXmTX
90hy94ataBjhGxsCfiOah2rtAeOo0THB4bP3z6u1BGqf2lbNRNZLz5rOyhcmqUf1CP0
363E7Vrr5vYtAgOYJ97DERsgy7RM3zMujRXhs3+p+GFcbEeCddycxgt3jCchZDgThm8
2SGro7/wmyCPEg75qhih+Eih52w/Qop/9PFzyyXBGXaZWvkO5uebdzSjce5xN9BgeoC
KwSwKtC898ZSQlO+o953TaJpby/iccjRPJwZjzCFrmtUBbFPYkzhCKuwjhn7TyXzYaV
fy5mrI2xoREvB44SwP0liRsagQ==
------------------------------License------------------------------

**Comment by LFTEC** [comment]
AV4sh5cxWaQXVpLrVSEUS2dJQkIS/g3IvJ7IUdJUKmqZn9qv0s3xS+9timVm6wQdZLRpkpfqoN4G/hSYCc/4Sx/BuUdR0WsbqwHaxQu3UOYCUMq/Uc42O8/VBY1LHqod4nis90Ao9GaMCUODN3Hei3BhV5/kCVcp0HOM3IiHAfTBiRjs8fvlLZWnNERWNeqigRo5Jjw8ZHAUgNFg2du6iTFfMxZmaoCTaH4hxxyC9LZuDFPvPBNSvgrM7MB2PzboDzhXO67EzbYEcymGPM7FnD3najQpQdoIkskTYVF9tFgvVwhidYnuh0/obd+WU81n5fD1Wo5aZjUD0PYbC+j+dxlt79vd5kger2MdNJtSvH8lrTiPSIuFS9gMiSFd1pgxPd15kIo1xVkytPfXeFwgCTf/gE6GHYx+uu4btLtquTx82WaVYNMqnrYyFt9tcPQYn4qjVomWTw+iyB+TQD/aH7jMQV5DQlm7b/uaSVJQJ2DhyG1Tr3q6JDMlNS4m3fp8OUrr82BLTjT3w48KDlFKl2LCk4wtVAmlWi43NzDX7rlU9WUpjGAdBz3LaZT+xppw+8VYusIDg8Bmhv9mKP7UcMPdJHQYHuIbykYe2aCnYZ3NCe9npBTUcApTbgJVgmJ4whwQC+u0c6hmPJdGYoWbD04JI1Ej4jxHc1UaMfVxzHuEiOaIJlvWNzOrIkKQ55acZ5+GA0iCy/itKcQRiatrAP3mlmvOB2jShqO80ZjOhSJg9sWrJwcdzA6KJqM094c7tay+diOMWIRJwrwCfmuxwuOiv1NWLb+EYaTa57W2jFW8WhHUHQARGTyx28Zy2CSHFMcok/V8HE7lXsOrKV2mz1qfKfNtkOeB8H0FaUeyBQ0wDq61sEzf2HCY/l83yyDH+yDyg0c4HCkxykfvbviCnIythKvcS097ZgkxqjjEgVw6mIRWzEKNiPuPq070wfnRgG3HvfGOPGj4vnMyxlze0gKAPP3+0mjW3e0469G0QBCmZAewHaETn8Mr0wX0jhj0Dc0nhmGytwzvsj5uamiOaRfO4tn6pFjAVFXC3QuLrCDfCkRyxUvd5FsFgBPMzo1YUW8aLRLFOqpIurZNVI50EMpxzh0C7mnvWPNkeFJcnqz27qa/9VfvoSYuBvxrEgRL83LKyRxKAYS88bc/J0OIgDLV6rGsH2VE3papDYb1b57Ao68Tkmb6ejAW9WvmoznIGUFLu2iWdP48vI8Gw2lyZIhH34+uWAZ5dlkLeBVyLR53hIDJYzEb0g7e9638m34ZnZpIPp8gJd7HlyIzQqRnGnJhcCJgtU9lMaDLMJ1GkDsZCZ9kgEEYhQsOvmmvFSXSIF1VjaMmKPntjbZT76UzgA==

**Reply by user2027** [reply]
大佬好，发现刷课时是隔一段时间得一点分，而课程考试 是一分钟就全部答完了，有没有考虑课程考试设置个延时呀，答完一题 过一会再答另一个课程

**Reply by LFTEC** [reply]
考试是1s回答一道题，考试结束后无论是否通过都间隔1s再考。必修课随机选课考试。选修课根据你本周随机学习的课程进行考试
